### DISCUSSION TITLE
Welcome to parallel-hashmap Discussions!

### DISCUSSION THREAD

**Discussion by greg7mdp** [greg7mdp/parallel-hashmap#77] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by lifengyu2005** [comment]
We used 16 submaps to insert 2 * 1024 * 1024, and found that the loading rate was about 50%; We use a fixed number of items and do not insert items dynamically in the middle, but expect a higher load rate, otherwise memory will be wasted seriously; Are there any parameters that can be adjusted ？
